src/pybind/mgr/openattic/module.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+"""
+openATTIC mgr plugin (based on CherryPy)
+"""
+
+import cherrypy
+from cherrypy import tools
+
+from mgr_module import MgrModule
+
+"""
+openATTIC CherryPy Module
+"""
+
+
+class Module(MgrModule):
+
+    """
+    Hello.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(Module, self).__init__(*args, **kwargs)
```

Comment:
I currently get a `W0235:Useless super delegation in method '__init__'` - not sure if that can be ignored?